tests/rt.conf

```diff
@@ -205,6 +205,9 @@ RUN     | hafs_regional_storm_following_1nest_atm_ocn
 RUN     | hafs_regional_storm_following_1nest_atm_ocn_wav                                                                         | - jet.intel                             | fv3 |
 RUN     | hafs_global_storm_following_1nest_atm                                                                                   | - jet.intel                             | fv3 |
 
+COMPILE | -DAPP=HAFSW -DMOVING_NEST=ON -DCCPP_SUITES=FV3_HAFS_v1_thompson_noahmp_nonsst,FV3_HAFS_v1_thompson_noahmp,FV3_HAFS_v1_thompson_nonsst,FV3_HAFS_v1_thompson,FV3_HAFS_v1_gfdlmp_tedmf_nonsst,FV3_HAFS_v1_gfdlmp_tedmf,FV3_HAFS_v1_thompson_tedmf_gfdlsf -D32BIT=ON -DSUPAFAST=ON |                | fv3 |
+RUN     | hafs_regional_storm_following_1nest_atm_ocn_wav_supafast                                                                | - jet.intel                             | fv3 |
+
```

Comment:
@SamuelTrahanNOAA, @junwang-noaa, I am not sure if we would prefer to add a new HAFS regression test (trying to avoid adding more regression tests if not absolutely needed.). Alternatively, we can just use the hafs_regional_storm_following_1nest_atm_ocn_wav RT but test it with the SUPAFAST/PRODUCTION/PROD build option instead.

Reply:
Yes, I will make that change. It means this PR will be answer changing.

Reply:
Testing this now.

Reply:
Perhaps we should run all of the HAFS regression tests with this build option?

Reply:
@SamuelTrahanNOAA, Thanks! Currently, we can just change to use this SUPAFAST/FAST/PRODUCTION/PROD option for this hafs_regional_storm_following_1nest_atm_ocn_wav RT. Later on, we can consider using it, especially after passing more rigorous tests.

Reply:
My current test only uses it for hafs_regional_storm_following_1nest_atm_ocn_wav